application/src/test/java/org/opentripplanner/updater/vehicle_rental/datasources/gbfs/v2/GbfsFeedMapperTest.java

```diff
@@ -116,6 +121,32 @@ void getOneVehicleTypeOfDuplicatedVehicleTypes() {
     assertEquals(1, vehicleTypes.size());
   }
 
+  @Test
+  void testClientExceptionRethrowsAsUpdaterConstructionException() {
```

Comment:
Is this the correct place for this? There is also `GbfsFeedLoaderTest` which seems to have more tests similar to this. It has some real integration tests which are disabled (because we try to avoid those), but we usually avoid mocking things as well. I'm not sure what would be the best approach here. @leonardehrenfried has created a submodule for real integration tests which are run in CI if the pr has certain label. I wonder if that could be expanded with realtime updater related tests. Although, in this case a real integration test might not make any sense. I can bring this up in the dev meeting tomorrow.

There are still the tests from https://github.com/opentripplanner/OpenTripPlanner/pull/4826, I'm not sure what we should do about those and if they really test anything.